applications/app/views/fragments/crosswords/crosswordContent.scala.html

```diff
@@ -6,51 +6,54 @@
+                                <div class="crossword__container__grid-wrapper">
+                                    <noscript>@crosswordPage.svg</noscript>
+                                </div>
 
-                                    <noscript>
-                                        <div class="crossword__clues">
-                                            <div class="crossword__clues--across">
-                                                <h3 class="crossword__clues-header">Across</h3>
-                                                @crosswordEntries(crosswordPage.crossword.entries.filter(_.direction == "across"))
-                                            </div>
+                                <noscript>
+                                    <div class="crossword__clues">
+                                        <div class="crossword__clues--across">
+                                            <h3 class="crossword__clues-header">Across</h3>
+                                            @crosswordEntries(crosswordPage.crossword.entries.filter(_.direction ==
+                                            "across"))
+                                        </div>
 
-                                            <div class="crossword__clues--down">
-                                                <h3 class="crossword__clues-header">Down</h3>
-                                                @crosswordEntries(crosswordPage.crossword.entries.filter(_.direction == "down"))
-                                            </div>
+                                        <div class="crossword__clues--down">
+                                            <h3 class="crossword__clues-header">Down</h3>
+                                            @crosswordEntries(crosswordPage.crossword.entries.filter(_.direction ==
+                                            "down"))
                                         </div>
-                                    </noscript>
-                                </div>
+                                    </div>
+                                </noscript>
                             </div>
                         </div>
                     </div>
-                    @if(adsEnabled) {
-                        <div class="content__secondary-column hide-until-wide" aria-hidden="true">
-                        @fragments.commercial.standardAd("right", Seq("mpu-banner-ad"), Map()))
-                        </div>
-                        <div class="crossword-banner hide-until-tablet hide-from-wide" aria-hidden="true">
-                        @fragments.commercial.standardAd("crossword-banner", Seq("crossword-banner"), Map())
-                        </div>
-                    }
                 </div>
+                @if(adsEnabled) {
+                <div class="content__secondary-column hide-until-wide" aria-hidden="true">
+                    @fragments.commercial.standardAd("right", Seq("mpu-banner-ad"), Map())
```

Comment:
There are now only two closing brackets after `Map(` instead of three. The rest of this file are Prettier changes